### PR TITLE
Bodoni Moda: Version 2.3 added

### PR DESCRIPTION
**Pull request by EwonRael**
Taken from the upstream repo https://github.com/indestructible-type/Bodoni/tree/google at indestructible-type/Bodoni@4a46643

### CODE REVIEW

**Comment by RosaWagner** [comment]
Thanks @EwonRael for the PR ! Bodoni was actually on my to-do list to onboard it on GF. 

There are some STAT and naming issue. I would suggest to follow the process that we have in [this repo](https://github.com/Omnibus-Type/Ballet) which have an optical size axis, and adapt of course the gen_stat.py and the design space. This one doesn't have a weight axis so no fvar instances, but yours are well set with just the weight, just the master used as font origin can be called "BodoniModa-Regular" (and not "BodoniModa11-Regular") and the output path for the VF should be …/BodoniModa[opsz,wght].ttf. Same for the fvar instances of the VF: familily name= Bodoni Moda, style name= Bold. No point size.

Then, for the static instances, you can append the point size as you already do, but with "pt" appended like: BodoniModa11pt-Regular.ttf

2 separate VF for upright and italic is great. So no ital axis in the designspace, but definetely one set up in the STAT. Actually you can take [this script](https://github.com/googlefonts/literata/blob/master/sources/gen_stat.py) or study Literata repo in general.

Tell me if something is unclear !

**Comment by EwonRael** [comment]
Hey @RosaWagner thanks for looking things over. Glad to know that Bodoni Moda hasn't been forgotten. 

The Ballet family is gorgeous. I couldn't find anything with a point size axis in production for reference so I'm glad you directed me towards that.

Happy to take the point size out of the variable builds name. Someone at some point I feel like told me to do that. I will have to go digging to figure out if that was the case.

I think everything looks clear to me. I will have to look it over more carefully when I have more time. I'd love to get more self-sufficient at this kind of thing. I love making fonts, but if I figure out how to do this kind of thing better maybe I can help play a more supportive role in other people's projects. That would be cool. Just if I could get more involved in general.

Maybe not the place to ask but how are things with the Jost? I tried to push changes. Not reflected on GF yet.

I'll get back to you

**Comment by RosaWagner** [comment]
>Happy to take the point size out of the variable builds name. Someone at some point I feel like told me to do that. I will have to go digging to figure out if that was the case.

For the statics the point size is supposed to be in the family name. But since the variable get all of them, no need to append it.

> Maybe not the place to ask but how are things with the Jost? I tried to push changes. Not reflected on GF yet.
Jost is in sandbox so it should be in production next week if everything goes according to plan 👍

**Comment by RosaWagner** [comment]
Please, also add some promotional pictures for GF twitter in your repo (in a "documentation" folder for example). We need 2-3 images with a nice twitter-friendly ratio. Video is also accepted.

**Comment by EwonRael** [comment]
I updated the font files to incorporate the changes. Let me know if it looks good.

When I first announced Bodoni* at indestructible type* I [made a little video for twitter](https://twitter.com/indestructype/status/1088880654043951104?s=20). Would something like that be good (with Bodoni Moda not bodoni etc.)? 

I think the little song is quite cute but maybe something without sound would be better?

Let me know!

File: ofl/bodonimoda/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  copyright: "Copyright 2020 The Bodoni Moda Project Authors (https://github.com/indestructible-type)"
+}
+fonts {
+  name: "Bodoni Moda"
+  style: "italic"
+  weight: 400
+  filename: "BodoniModa-Italic[opsz,wght].ttf"
+  post_script_name: "BodoniModa11-Italic"
+  full_name: "Bodoni Moda Italic"
+  copyright: "Copyright 2020 The Bodoni Moda Project Authors (https:github.comindestructible-type)"
```
Comment (RosaWagner) [inline ofl/bodonimoda/METADATA.pb:22]: I updated the metadata file which shows that the regular is good, while the italic still have the wrong postscript name and a typo in the copyright URL (https://)

**Comment by RosaWagner** [comment]
For the instances (fvar and exported statics), we also need SemiBold (600)

**Comment by EwonRael** [comment]
I think I fixed all the metadata stuff in the italic. Weird that it lost all the "/"s in the copyright.

I can have it build a semibold... For some reason this feels in conflict with the artistic vision. Not the hill I want to die on. Are you also wanting an 800 weight?

**Comment by RosaWagner** [comment]
For the semibold, @davelab6 will explain it better than me. We need be able to cover a full range of weight between the 2 given extremes (https://github.com/googlefonts/fontbakery/issues/3031), so no need to add the 800. In GF API, the instances are generated from the variable font, and they it will display a SemiBold in any case. We need the exported statics for the ttf hinting, if I understood correctly… so if the 600 is missing, then the statics are not so useful.

**Comment by EwonRael** [comment]
I made the semibold and extrabold.

Let me know if everything looks good, and let me know about the music twitter question. I wrote that song and am quite proud of it.

**Comment by RosaWagner** [comment]
@EwonRael can you also reexport the variable fonts with semibold and extrabold instances?

**Comment by RosaWagner** [comment]
please also do `gftools add-font ofl/bodonimoda` to update metadata.pb

**Comment by RosaWagner** [comment]
And the music is indeed lovely.

**Comment by RosaWagner** [comment]
There is still something wrong with the STAT table, and it is probably coming from your build script. You are using fix-vf-meta, statmake and the stat.py script. 
- fix-vf-meta > we use it for fonts with only one axis
- statmake > we don't use it anymore

Post-process of your variable fonts should look like this: 

```
echo "Hot Fixes for Variable Font"
vfs=$(ls ../fonts/variable/*.ttf)
for font in $vfs
do
	gftools fix-dsig --autofix $font
	gftools fix-nonhinting $font $font.fix #we dont hint new variable fonts
	mv $font.fix $font
	gftools fix-unwanted-tables --tables MVAR 
done
rm ../fonts/variable/*gasp*
python3 ../sources-GF/Designspace/stat.py
```

Fontbakery also report an issue with your **GDEF table**. 

Make sure:
- only the comb accents have anchors and 0-width
- the accents (without comb in the name) don't have 0 width

**Comment by EwonRael** [comment]
made fixes hopefully that's the last round

[video up on repo](https://github.com/indestructible-type/Bodoni/blob/google/documentation/BodoniModa.mp4)

**Comment by RosaWagner** [comment]
New PR : #2833

**Pull request closed without merging**